.github/workflows/ocwm-issue-collector.yml

```diff
@@ -111,6 +113,8 @@ jobs:
                     let json_text = JSON.stringify(body.substring(0, startIndex - 1) + `\n| ${url} - ${title} | ${author} |\r\n` + body.substring(startIndex, body.length));
                     body = JSON.parse(json_text);
                     changesFlag = true;
+                    // Remove the 'agenda' label from the added discussion
+                    await mygithub.request(`DELETE /repos/${process.env.OWNER}/${repositories[r]}/discussions/${discussions[i].number}/labels/${appendLabel}`);
```

Comment:
The api necessary for managing GitHub discussions uses GraphQL instead rest, so you well need to change your code:

https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions#deletediscussioncomment

Reply:
Yeah my code uses exactly the same thing you asked for @benjagm 
RemoveLabelsFromLabelableInput!{
                        removeLabelsFromLabelable(input:$input){
                          clientMutationId
                    }
              }

and i added the REST api call to get the label.